### PR TITLE
Chore/#22 docs examples

### LINKED ISSUE #22: add an example data-resource structure to "usage" page in documentation
think this will help to get familiar with the usage more easy.

### PR DESCRIPTION
**Pull request by ctmm**
closes #22 

#167 Should be merged first

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -29,67 +38,4 @@ const resourceLoader = new ResourceLoader({
 <div data-anything="[]"></div>
```
[the remark this answers is not in the document]
Comment (ctmm) [reply docs/usage.md:38]: Absolutely. 

I didn't put them it this example as it's part of the getting started guide, and I wanted to keep it as simple as possible.

This fact that `data-init`and `data-init`are also completely rewritable is outlined on the API guide

File: docs/initialize.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      // options = any JSON parsable content you set on data-options attribute
+      …
+    }
+}
+
+registerScript(
+  // your plugin's class
+  MyPlugin,
+  // the name you want to set your plugin to be used via the data-init attribute
+  'MyPlugin'
```
[the remark this answers is not in the document]
Comment (ctmm) [reply docs/initialize.md:28]: I didn't even think of that possibility but I assume that modules which do not expose any Class should "self-initialize" on load.

Do we need this feature for a particular project or can it be regarded as a "nice-to-have"?

[the remark this answers is not in the document]
Comment (ctmm) [reply docs/initialize.md:28]: If this is a necessary feature, please open an issue to centralise the discussion there.

[the remark this answers is not in the document]
Comment (timomayer) [reply docs/initialize.md:28]: @ctmm maybe you do a quick research over some projects using v1 and v2 of the user i guess there are a lot of files not exposing a class and are not self initializing, i ll open an issue.

**Pull request merged**